bin/MessagesBundle_en.properties

```diff
@@ -114,17 +114,33 @@ extFCJ_c=Activate FidoCadJ extension while copying elements
 Restart=Startup
 Drawing=Drawing
 FidoCad=FidoCadJ extensions
+
+Theme_management=Theme and color management
```

Comment:
Put all the resources at the end of the file, so to keep roughly the same order in all files.